### ISSUE TITLE
Alt text and styling missing for images

### ISSUE THREAD

**Issue by ruchamahabal** [frappe/wiki#159]
Previously wiki used to set alt text from image filenames. Now alt property is not set

<img width="1326" alt="image" src="https://github.com/frappe/wiki/assets/24353136/a36eac75-1ec1-44b7-8d5e-122453dca72f">

Also, the images used to have a `screenshot` class adding border and border-radius for a cleaner look

<img width="1440" alt="image" src="https://github.com/frappe/wiki/assets/24353136/faf99ca0-fabb-42e8-a1f6-ebcecca5aa02">

Using custom js to add the class to images for now: https://frappehr.com/docs/en/employee
But can it be handled in the core?

**Comment by BreadGenie** [comment]
Added in 725c526609ff8417a73451f624a4fdd4f72fe0d0, 0d6643176742c03dc1ad0bf360a579d813dc1945.
I'm not sure using the file text as alt is a great option but better than nothing ig.
